src/views/docs-view/__tests__/multiple-remark-plugins.test.ts

```diff
@@ -0,0 +1,30 @@
+import remark from 'remark'
+import { cachedGetProductData } from 'lib/get-product-data'
+import { rewriteTutorialLinksPlugin } from 'lib/remark-plugins/rewrite-tutorial-links'
+import remarkPluginAdjustLinkUrls from 'lib/remark-plugin-adjust-link-urls'
+import { getProductUrlAdjuster } from '../utils/product-url-adjusters'
+
+describe('multiple remark plugins', () => {
+	let waypointProduct
+	let productUrlAdjuster
+
+	beforeAll(() => {
+		waypointProduct = cachedGetProductData('waypoint')
+		productUrlAdjuster = getProductUrlAdjuster(waypointProduct)
+	})
+
+	test('rewriteTutorialLinksPlugin, then remarkPluginAdjustLinkUrls', async () => {
+		const linkToProcess = '[Waypoint](/waypoint)'
```

Comment:
❓ are we still running this plugin tutorials content? I can't remember.

Reply:
`DocsView` still uses `rewriteTutorialLinksPlugin`, is that what you're asking?

Reply:
Oh no I was asking if we still run this plugin on tutorials content, but I remember [now that it was removed!](https://github.com/hashicorp/dev-portal/pull/1319)